packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec

```diff
@@ -6,17 +6,17 @@
 %global gem_name hammer_cli_katello
 %global confdir hammer
 
-%global release 3
-%global prerelease .pre.master
+%global release 4
+%global prerelease pre.master
 
 Summary: Katello command plugin for the Hammer CLI
 Name:    %{?scl_prefix}rubygem-%{gem_name}
 Version: 0.15
```

Comment:
Also moved to 0.16: https://github.com/Katello/hammer-cli-katello/commit/f1a3f73d940b6c3cc604af2bb640842a61a07f79